lib/datadog/grok/src/filters/array.rs

```diff
@@ -0,0 +1,187 @@
+use crate::{
+    ast::{Function, FunctionArgument},
+    parse_grok_rules::Error as GrokStaticError,
+};
+use bytes::Bytes;
+use nom::{
+    branch::alt,
+    bytes::complete::{tag, take, take_until},
+    character::complete::char,
+    combinator::cut,
+    combinator::map,
+    multi::separated_list0,
+    sequence::{preceded, terminated},
+    IResult,
+};
+use std::convert::TryFrom;
+
```

Comment:
Tiniest of nits, this blank line probably doesn't need to be here?

Reply:
Right, updated.